EIPS/eip-7733.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: 7733
+title: Deactivate EIP-158
+description: Deactivates EIP-158 in order to avoid conflicts in a context in which statelessness and EIP-7702 are active
+author: Guillaume Ballet (@gballet)
+discussions-to: https://ethereum-magicians.org/t/eip-7733-eip-158-deactivation/20445
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-07-02
+requires: 158, 7702
+---
+
+## Abstract
+
+Deactivate [EIP-158](./eip-158.md).
```

Comment:
You should expand a little bit on your abstract. This certainly satisfies the terse requirement, but doesn't give the reader any information about the content of this document. Would you mind adding a sentence explaining, at a high level, what deactivating EIP-158 entails?